### DISCUSSION TITLE
[Question]: Is there a way to hide the file upload button when it's not a vision model?

### DISCUSSION THREAD

**Discussion by makeneko** [danny-avila/LibreChat#1673] [Q&A]
### What is your question?

Is there a way to hide the file upload button when it's not a vision model?

### More Details

In my environment, I only use Azure GPT-4 and GPT-3.5-turbo. I do not use vision models. However, the file upload button is displayed, and uploading files results in an error.

It seems that conditions like isVisionModel are not used in AttachFile.tsx.
https://github.com/danny-avila/LibreChat/blob/main/client/src/components/Chat/Input/Files/AttachFile.tsx

### What is the main subject of your question?

UI

### Screenshots

![image](https://github.com/danny-avila/LibreChat/assets/551549/fe3a8282-8c93-4072-ab6a-5878e95dc191)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Currently no but I hope to add deeper configuration for things like this as it pertains to files. Of course you can manually remove it in the code but a configurable manner is planned.

**Comment by makeneko** [comment]
@danny-avila 
Thank you for your prompt response.
I understand that it cannot be set under the current conditions.

I feel that it is a very wonderful project.
I am supporting you.
